gamemodes/terrortown/gamemode/shared/sh_init.lua

```diff
@@ -493,6 +497,7 @@ COLOR_ORANGE = Color(250, 100, 0, 255)
 COLOR_OLIVE = Color(100, 100, 0, 255)
 COLOR_BROWN = Color(70, 45, 10)
 COLOR_LBROWN = Color(135, 105, 70)
+COLOR_CUSTOMGRAY = Color(91, 94, 99, 255)
```

Comment:
please rename this. Something like `LGRAY` or `DGRAY` or so

Reply:
It's nether light nor dark, it's a custom gray you defined but not localized / globalized :D

Reply:
I see what you're doing here, but I don't like "custom" because everything is custom. Maybe call it `COLOR_WARMGRAY`